contracts/IbetShare.sol

```diff
@@ -100,6 +118,18 @@ contract IbetShare is Ownable, IbetStandardTokenInterface {
     // イベント：減資
     event Redeem(address indexed from, address indexed target_address, address indexed locked_address, uint256 amount);
 
+    // イベント：移転承諾要否変更
+    event ChangeTransferApprovalRequired(bool required);
+
+    // イベント：移転申請
+    event ApplyForTransfer(uint256 indexed index, address from, address to, uint256 value, string data);
+
+    // イベント：移転申請取消
+    event CancelTransfer(uint256 indexed index, address from, address to);
+
+    // イベント：移転承認
+    event ApproveTransfer(uint256 indexed index, address from, address to, string data);
```

Comment:
[imo]`Transfer`イベントと合わせると、移転数量があると使いやすいですかね？（indexあるんで引けますが...）

Reply:
I thought it was necessary for partial approval, but since it was full approval (or full cancellation), I thought it was unnecessary.
> 部分的に承認する場合は必要と思ったのですが、全承認（or全キャンセル）なので、不要と考えました。